server/cache.go

```diff
@@ -225,6 +225,13 @@ func (r *regionsInfo) randFollowerRegion(storeID uint64) *regionInfo {
 	return randRegion(r.followers[storeID])
 }
 
+func (r *regionsInfo) randomRegion() *regionInfo {
+	for _, region := range r.regions {
```

Comment:
do you mean if the regions is not changed, every for in range will get a random region? 

maybe we should add a test to verify it.